bundles/org.openhab.binding.clementineremote/README.md

```diff
@@ -0,0 +1,334 @@
+# ClementineRemote Binding
+
+<img style="float: right; margin: 0 0 5px 5px" src="doc/screenshot.png" alt="Screenshot" />
+This binding shall bring the benefits of over-the-network control of the [Clementine-Player] to OpenHab!
```

Comment:
OpenHab => openHAB
If there are more, please correct all of them.

Reply:
Fixed.